dialog-polyfill.js

```diff
@@ -483,4 +483,12 @@
   window['dialogPolyfill'] = dialogPolyfill;
   dialogPolyfill['forceRegisterDialog'] = dialogPolyfill.forceRegisterDialog;
   dialogPolyfill['registerDialog'] = dialogPolyfill.registerDialog;
+
+  if ( typeof module === 'object' && typeof module.exports === 'object' ) {
```

Comment:
So I'm concerned compiling this with Closure (which is optional, but works well) might convert `module` to a tiny variable name. But I'm happy to fix that later.